Source/ProjectFirma.Web/Views/Reports/Edit.cshtml

```diff
@@ -37,7 +38,14 @@
                 @Html.LabelWithSugarFor(x => x.DisplayName)
             </div>
             <div class="col-md-10">
-                @Html.TextBoxFor(x => x.DisplayName, new { @class = "form-control" })
+                @if (ViewDataTyped.ReportTemplate != null && ViewDataTyped.ReportTemplate.IsSystemTemplate)
+                {
+                    @ViewDataTyped.ReportTemplate.DisplayName
```

Comment:
The edit actually fails here because "DisplayName" is not part of the post back. We either need to change this to be a readonly text box, or add a hidden field with the display name.